README.md

```diff
@@ -1,19 +1,19 @@
-# Sequence [![Build Status](https://travis-ci.org/revinate/sequence.svg)](https://travis-ci.org/revinate/sequence)
```

Comment:
Could we potentially get, either in this README or the PR notes, a way of quickly installing and testing this via Docker? Will be cleaner and simpler to test than installing PHP 8.1 on folks' machines.

What about instructions to start a docker container from a php-8.1 image, log into the container, clone down the repo, install composer / run composer install, and run `phpunit`? Could maybe also build this into a `Dockerfile`, but I'm not sure about getting the repo files into there without mounting a volume. Let me know what you think, maybe this isn't too much work to do.

I also want to make sure we test this on php 8.0 - that's actually more important, as we don't want to break production before we roll out to 8.1. Using docker would make it easy for us to base the image on `php-8.0` as well to test that.

Reply:
@LuisMejiaF added a Dockerfile (thanks!) 5e9499649d949ee77cafd18da7acc6145214cdeb
 I've updated the testing notes with the instructions. For testing in our app I will open a PR with the instructions as well